.dockerignore

```diff
@@ -0,0 +1,16 @@
+README.md
+LICENSE
+.gitignore
+.yarnrc
+package.json
+package-lock.json
+yarn.lock
+version.txt
```

Comment:
Are you still using this file? We can probably just delete it right?

Reply:
🤷🏻

Reply:
definitely not using version.txt so I removed it.